todo/handler.go

```diff
@@ -74,5 +85,8 @@ func (h *TodoHandler) DeleteTodo(c *fiber.Ctx) error {
 		return c.Status(fiber.StatusInternalServerError).JSON(fiber.Map{"error": err.Error()})
 	}
 
-	return c.SendStatus(fiber.StatusNoContent)
+	return c.Status(fiber.StatusNoContent).JSON(fiber.Map{
+		"status":  "success",
+		"message": "Todo deleted successfully",
```

Comment:
data bir var bir yok. belli bir formatta değil. ayrıca bunu bir method haline getirmen lazım sen bu formatta gideceğini biliyorsun ama başka biri geliştirme yapsa formatı bilmeyeceği için format dışına çıkabilir yada typo hatası yapabilir. bu yüzden bu format işini bir method yapmalı.